core/feconf.py

```diff
@@ -499,6 +499,8 @@ def get_empty_ratings() -> Dict[str, int]:
 DATAFLOW_TEMP_LOCATION = 'gs://todo/todo'
 DATAFLOW_STAGING_LOCATION = 'gs://todo/todo'
 
+# The path to the package contains version number but we don't increment
```

Comment:
Add TODO perhaps, with issue, per our discussion in chat.

Reply:
Done.